### ISSUE TITLE
Plots anpassen

### ISSUE THREAD

**Issue by AleksandarRa** [AIS-22/UNI-AIS-MediaDataFormats#82]
### Accuracy plot:
+  schlage vor das in 2 plots zu teilen; der erste zeigt alle mit transfer-rearning; der zweite zeigt transfer learning 10, 25, 40, 100 sowie mixed self-trained und mixed pre-trained; bitte mit farben aufpassen - besonders die mit blau kennt man fast nicht auseinander (5 und mixed) .......

### Fontsize anpassen von folgenden Plots:
+ Accuracy plot
+ confusion matrix plot
+ loss function plot
+ quality plot
+ psnr plot
+ target filesize plot
